src/dcmp/dcmp.c

```diff
@@ -1768,7 +1861,8 @@ static void dcmp_disjunction_print(
 static int dcmp_disjunction_match(
     struct dcmp_disjunction* disjunction,
     strmap* map,
-    const char* key)
+    const char* key,
+    int is_src)
```

Comment:
I believe that `stdbool.h` is included in this file. If `is_src` is being used as a boolean, consider using the `bool` type.